CONTRIBUTING.md

```diff
@@ -115,31 +126,52 @@ If you are working on windows, check out [`jekyll` docs](https://jekyllrb.com/do
 Take a look at an existing [demo file](https://github.com/fabricjs/fabricjs.com/blob/gh-pages/posts/demos/_posts/2020-2-15-custom-control-render.md).
 Create a new file in the same directory (`posts/demos/_posts`) and you're good to go.
```

Comment:
when did we deprecate adding a demo and checking that fabricjs.com kitchensik?
Maybe those aren't the best way to do things, but aren't out of the way